public/index.html

```diff
@@ -2,17 +2,19 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <!--suppress HtmlUnknownTarget -->
```

Comment:
What's this used for?

Reply:
suppress ide warning check

Reply:
Would it be possible to set this up in a config file which can be ignored by .gitignore?

Reply:
I don't feel so. This is autogenerated by IDEA

Reply:
then one of the approaches may put this file into .gitignore.
It will be confusing to have a ide specific feature implemented in the code other than configuration files